README.md

```diff
@@ -1,66 +1,83 @@
 
-This should install all of our project dependencies from npm into a local `node_modules` folder.
+  The phone numbers should be in
+  [E.164 format](https://www.twilio.com/help/faq/phone-numbers/how-do-i-format-phone-numbers-to-work-internationally).
 
-### Configuration
+  Run `source .env` to export the environment variables.
 
-Next, open `config.js` at the root of the project and update it with values from your environment and your [Twilio account](https://www.twilio.com/user/account/voice-messaging). You can either export these values as system environment variables (this is the default setup), or you can replace these values with hard-coded strings (be careful you don't commit them to git!).
+1. Navigate to the project directory in your terminal and run:
 
-### Running the Project
+  ```bash
+  $ npm install
+  ```
 
-To launch the application, you can use `node .` in the project's root directory. You might also consider using [nodemon](https://github.com/remy/nodemon) for this. It works just like the node command, but automatically restarts your application when you change any source code files.
+  This should install all of our project dependencies from npm into a local `node_modules` folder.
 
-```bash
-npm install -g nodemon
-nodemon .
-```
+1. To launch the application, you can use `node .` in the project's root directory. You might also consider using [nodemon](https://github.com/remy/nodemon) for this. It works just like the node command, but automatically restarts your application when you change any source code files.
+
+  ```bash
+  $ npm install -g nodemon
+  $ nodemon .
+  ```
 
 ### Docker local development
 
-First, install [Docker](https://www.docker.com/) and [docker-compose](https://docs.docker.com/compose/install/).
+1. Install [Docker](https://www.docker.com/) and [docker-compose](https://docs.docker.com/compose/install/).
+
+1. Make sure you performed step `2` on the previous section.
 
-Then set the following environment variables in your terminal session. You can find these values at https://www.twilio.com/user/account/voice and at https://dashboard.authy.com.
+1. Finally, run the following commands to start your Docker containers:
 
-- `TWILIO_NUMBER`
-- `TWILIO_ACCOUNT_SID`
-- `TWILIO_AUTH_TOKEN`
-- `AGENT_NUMBER`
+  ```
+  $ docker-compose up -d
+  ```
 
-Finally, run the following commands to start your Docker containers:
+  Warning: If you previously ran  ```npm install``` locally, the node_modules folder will conflict with the file structure of the container when you run the above command.
```

Comment:
Those spaces are indenting the paragraph as it's in an ordered list, it would look awkward without don't you think?